build/server-image/kube-apiserver/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG arch
+ARG base_image
+
+FROM --platform=linux/${arch} debian
```

Comment:
we should probably use our own debian image from k8s.gcr.io?

I think this is a reasonable starting point, but a future trade-off consideration might also be having a "setcap" image (not the go-runner) that we use for building, so we can pull the image once and not need to live apt-install during `make quick-release` etc... (less dependency on external infra)

Reply:
basically https://github.com/kubernetes/kubernetes/pull/96134#discussion_r517149798 just not in the runtime image still.

Reply:
Can we do that as a follow up PR? It will require some coordination with the release team to push a new image that just contains setcap.

Reply:
that sounds reasonable, though I do think we should probably follow up with that sooner rather than later, I don't think we want to depend on live apt-get, none of our other builds do. (it also makes offline development problematic, which kubernetes is otherwise very good at today once you have the base images pulled)

Reply:
Sounds good, I'll do that as soon as this PR merges.